src/solve.jl

```diff
@@ -7,6 +7,11 @@ function DiffEqBase.__solve(jump_prob::DiffEqBase.AbstractJumpProblem{P},
     integrator.sol
 end
 
+# if passed a JumpProblem and no aggregator is selected use SSAStepper
+function DiffEqBase.__solve(jump_prob::DiffEqBase.AbstractJumpProblem; kwargs...)
+    DiffEqBase.__solve(jump_prob, SSAStepper(); kwargs...)
+end
```

Comment:
@ChrisRackauckas can you confirm this is an OK dispatch for me to add to enable 
```julia
sol = solve(jprob; kwargs...)  # equivalent to solve(jprob, SSAStepper(); kwargs...)
```

Reply:
Hmm, I guess this ignores when the JumpProblem is over an ODEProblem or such. Is there a way I can explicitly select the default auto-solver for a given problem type?